src/background/modules/ContextMenu.js

```diff
@@ -197,8 +234,9 @@ BrowserCommunication.addListener(COMMUNICATION_MESSAGE_TYPE.UPDATE_CONTEXT_MENU,
  * @returns {void}
  */
 export async function init() {
+    const platformInfo = await browser.runtime.getPlatformInfo();
     // Remove once https://bugzilla.mozilla.org/show_bug.cgi?id=1595822 is fixed
-    if (await isMobile()) {
+    if (platformInfo.os === "android") {
```

Comment:
Wait why not `isMobile`? What was bad about that?

Reply:
Nothing is bad about `isMobile`, I just needed to call `runtime.getPlatformInfo()` below (to determine the paste symbol), so no point in making that async call twice in the same function.

Reply:
Then I'd rather argue to add that to the env detector library.

Apart from that I would prefer code readability over preliminary performance optimisations, I.e. I'd rather prefer `isMobile` here then.

Reply:
I would argue that this is more readable, considering that `isMobile` does not actually check for all mobile OSs, just Android. This is especially important now that Safari on iOS supports Web Extensions (if someone ever wanted to updated this extension to support iOS).

Anyway, I can revert this optimization if you want...

Reply:
> isMobile does not actually check for all mobile OSs, just Android.

Oh yeah, then that's a bug of course. It should actually do that.

> This is especially important now that Safari on iOS supports Web Extensions (if someone ever wanted to updated this extension to support iOS).

Okay good, i was not aware of that. So yes, it should exclude that too _and_ then you can and should use that.

Reply:
OK, I reverted this optimization in https://github.com/rugk/unicodify/pull/72/commits/d629b8b4a7effcc841df56924a3a021d94ca04f0.